src/transformers/modeling_flax_pytorch_utils.py

```diff
@@ -224,15 +230,10 @@ def convert_pytorch_state_dict_to_flax(pt_state_dict, flax_model):
                 continue
 
             # also add unexpected weight so that warning is thrown
-            flax_state_dict[("params",) + flax_key] = (
-                jnp.asarray(flax_tensor) if not is_bfloat_16 else jnp.asarray(flax_tensor, dtype=jnp.bfloat16)
-            )
-
+            flax_state_dict[("params",) + flax_key] = np.asarray(flax_tensor)
```

Comment:
Why is this one set as a numpy array rather than jnp?

Reply:
oops yes I think it should be better